WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml

```diff
@@ -25,46 +25,51 @@
     <Style Selector="TextBox.Transparent">
       <Setter Property="Background" Value="Transparent" />
     </Style>
+    <Style Selector="Grid.CoinItemGrid">
+      <Setter Property="Background" Value="Transparent" />
+      <Setter Property="Height" Value="24" />
+    </Style>
+    <Style Selector="Grid.CoinItemGrid > :is(Control)">
```

Comment:
I require some explanations here.

Reply:
This is for centering the content on the direct descendants of `Grid.CoinItemGrid`